src/Racingcar.js

```diff
@@ -0,0 +1,46 @@
+import validation from './validation.js';
+import utils from './utils.js';
+import ERRORMESAGE from './errorMessage.js';
+import MAGICNUMBER from './magicnumber.js';
+
+export default class Racingcar {
+  #inputArr;
+  /**
+   *
+   * @param {Array} inputArr
+   */
+  constructor(inputArr) {
+    this.#validate(inputArr);
+    this.#inputArr = inputArr;
+  }
+
+  #validate(inputArr) {
+    if (!validation.isLengthBelowFive(inputArr))
```

Comment:
```suggestion
    if (validation.isMoreThanFive(inputArr))
```

유효성 검사에서 어떤 건 true일때, 어떤건 false일떄로 판단하고 있어서 혹시 일관성을 맞춰보시는건 어떨까요? 변수명과 조건을 조금 바꿔야겠지만, 이렇게 관리하는게 조금 더 낫지 않을까 싶어 제안드려요 !